### DISCUSSION TITLE
q-date event tooltip

### DISCUSSION THREAD

**Discussion by miguelrrsantos** [quasarframework/quasar#9715] [General - Components / Directives / etc]
Hi there.

I have a q-date where i setted up the :events prop to mark some dates as holidays. Is there a way to trigger a tooltip on that events days on calendar?

For now, the holidays are marked with a color below the day number but i want to achieve something that on hover that holiday/event day could trigger a tooltip with the event/holiday name.

Is that possible? And how?

This is my code for now that displays the events on calendar.
```
<q-date
    v-model="classAux.start_date"
    :events="highlightHolidays"
    mask="YYYY-MM-DD"
    @input="() => $refs.startDate.hide()"
/>
```

Best regards,
Miguel.

**Comment by ontwikkelfabriek** [comment]
@miguelrrsantos 

>Is that possible? And how?

Looking at the Api of the q-date component I don't see an 'easy' way of doing this. Having said that, did you know about the q-calendar extension for Quasar. I pretty confident that it will be able to show a tooltip (and a lot more):

https://quasarframework.github.io/quasar-ui-qcalendar/docs
https://github.com/quasarframework/quasar-ui-qcalendar

**Reply by miguelrrsantos** [reply]
Thanks for your reply. I'm using q-date inside input component. Does the calendar work the same way, so i can pick a date?

**Comment by pdanpdan** [comment]
If you don't use v2 then you can use my build:

https://pdanpdan.github.io/quasar-docs/vue-components/date#day-scoped-slot

In package.json replace quasar dependency with:
"quasar": "https://github.com/pdanpdan/quasar#quasar-pdan-v1.15.21-beta.2"

Then do a yarn or npm install and start the app again.

Please report if you find something not working.

**Reply by miguelrrsantos** [reply]
Thanks for your reply. I will have a look. Is there anything to know about using your build instead of the official quasar repository? Regards, Miguel.

**Reply by pdanpdan** [reply]
https://github.com/pdanpdan/quasar/releases
